icommons_ext_tools/cron.d/prod/qualtrics_user_update_cron

```diff
@@ -1,2 +1,2 @@
 # Update all required Qualtrics users at 4 am every Sunday
-0 4 * * 0 deploy /home/deploy/.virtualenvs/icommons_ext_tools/bin/python /opt/django/icommons_ext_tools/manage.py qualtrics --perform-full-update >> /var/opt/django/log/django-icommons_ext_tools.log 2>&1
+0 4 * * 0 deploy /home/deploy/.virtualenvs/icommons_ext_tools/bin/python /opt/django/icommons_ext_tools/manage.py qualtrics --map-qualtrics-ids-to-univ-ids >> /var/opt/django/log/django-icommons_ext_tools.log 2>&1
```

Comment:
This change needs to be done on the cron server, correct?

Reply:
Correct, this will update the cron job there on deployment